modules/postgresql/variables.tf

```
@@ -21,7 +21,19 @@ variable "project_id" {
 
 variable "name" {
   type        = string
-  description = "The name of the Cloud SQL resources"
+  description = "The name of the Cloud SQL Master instance"
```

nit: here and throughout

```suggestion
  description = "The name of the Cloud SQL master instance"
```